package.json

```
@@ -31,39 +31,46 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "9.0.4",
-    "@angular/cdk": "8.0.0",
-    "@angular/common": "^9.0.4",
-    "@angular/compiler": "^9.0.4",
-    "@angular/core": "^9.0.4",
-    "@angular/forms": "^9.0.4",
-    "@angular/material": "8.0.0",
-    "@angular/platform-browser": "^9.0.4",
-    "@angular/platform-browser-dynamic": "^9.0.4",
-    "@angular/router": "^9.0.4",
+    "@angular/animations": "13.0.0",
```

We should use ether a fix version number or dynamic, but do not mix both
```suggestion
    "@angular/animations": "^13.0.0",
```